projects/requisition-management/src/app/models/requisition/requisition.mapper.ts

```diff
@@ -51,6 +51,7 @@ export class RequisitionMapper {
             },
           },
           systemRejected: data.systemRejected,
+          systemRejectErrors: data.systemRejectErrors?.map(message => message.message),
```

Comment:
Because the message is an optional field, does it make sense to map the code here if the message is missing?
Otherwise you could remove the code from the interface if you never use it